UPGRADE-5.1.md

```diff
@@ -11,3 +11,8 @@ Routing
 -------
 
  * Deprecated `RouteCollectionBuilder` in favor of `RoutingConfigurator`.
+
+Validator
+---------
+
+ * When extending `DateValidator`, if you override the pattern, make sure you label your regex with day, month and year.
```

Comment:
that's not required as named arguments still generate positional arguments AFAIK

Reply:
Indeed